plugins/ar_spectral.py

```diff
@@ -202,24 +202,45 @@ def get_scan_res(self):
             yres = 1
         return int(xres), int(yres)
 
+    def _estimateCCDTime(self):
+        """
+        Estimate the time it will take for measurement at one ebeam position
+        """
+        # Each pixel = the exposure time (of the detector) + readout time +
+        # 10ms overhead + 20% overhead
+
+        binning = (self.binninghorz.value, self.binningvert.value)
+        res = [s / b for s, b in zip(self._detector.resolution.range[1], binning)]

```

Comment:
Is res the resolution value of the detector? Why don't we get self._detector.resolution.value? I thought it's calculated from the zip(detector.shape/binning). I know estimateAcquisitionTime is called before run acquisition but why don't we calculate the self._detector.resolution.value earlier? and if needed to be updated every time the binning changes, we can take care of it by subscribing to update the resolution.